Sources/Common/Core/ScalarsToColors/index.js

```diff
@@ -125,25 +125,26 @@ function vtkScalarsToColors(publicAPI, model) {
       newColors[count] = l;
       newColors[count + 1] = l;
       newColors[count + 2] = l;
-      newColors[count + 3] = convtFun(values[i + 1]);
+      newColors[count + 3] = convtFun(values[i + 1]) * alpha;
```

Comment:
@martinken do you agree with the logic of Alpha multiplication here? It seems that it was done for all other use case except this one. So we were wondering if it was missing or not.